scene/main/node.cpp

```diff
@@ -1322,6 +1323,51 @@ bool Node::can_auto_translate() const {
 	return data.is_auto_translating;
 }
 
+StringName Node::get_translation_domain() const {
+	ERR_READ_THREAD_GUARD_V(StringName());
+
+	if (data.is_translation_domain_inherited && data.is_translation_domain_dirty) {
+		const_cast<Node *>(this)->_translation_domain = data.parent ? data.parent->get_translation_domain() : StringName();
+		const_cast<Node *>(this)->data.is_translation_domain_dirty = false;
```

Comment:
```suggestion
		data.is_translation_domain_dirty = false;
```
This field is mutable, so the cast should be unnecessary.